openqa-label-known-issues

```diff
@@ -126,6 +126,7 @@ investigate_issue() {
     local id="${1##*/}"
     local reason
     local curl_output
+    echo "Requesting jobs/${id} via openqa-cli"
```

Comment:
Where will this message appear now?

Reply:
maybe in the gru log?

Reply:
Well, after my first comment you removed the `>&2`. Before it would appear in the gru journal. But we only want errors or warnings in the log.
What was the purpose of removing the `>&2`? Where should it appear?
If the message is supposed to be helpful it should show up somewhere?

Reply:
I added it because of the suggestion 
* Consider improving the scripting, error reporting, etc. e.g. reveal the URL/ API route

Reply:
Sure, but where does it appear? If we don't see it, we could also remove it again.
It could go to STDERR and land in the journal if you add `>&2`, but then it should only be written in case it fails.

Reply:
Actually, since this is openqa-label-known-issues: its output is filtered in our hook script:
https://github.com/os-autoinst/scripts/blob/master/openqa-label-known-issues-and-investigate-hook#L32
So it will never appear anywhere.

Reply:
I guess at least you can see it, if called by hand.